src/client/components/voyage/StartVoyage.js

```diff
@@ -24,17 +24,14 @@ class StartVoyage extends Component {
     departureTime: "",
     arrivalDate: "",
     arrivalTime: "",
-    isChecked: false,
-    draft_min: 0.0,
-    draft_max: 99.99,
+    isChecked: false
+  };
+
+  static defaultProps = {
```

Comment:
Input Field component set to use default props. Removed min/max from input props